code/game/supplyshuttle.dm

```diff
@@ -1416,7 +1416,7 @@ var/datum/controller/supply/supply_controller = new()
 		world.log << "## ERROR: Eek. The supply/elevator datum is missing somehow."
 		return
 
-	if(!is_admin_level(SSshuttle.vehicle_elevator.z))
+	if(!should_block_game_interaction(SSshuttle.vehicle_elevator))
```

Comment:
This should still be admin level, it's relevant to the elevator position proper

Reply:
What's wrong with having it like this? should_block_game_interaction still checks for it being an admin Z

Reply:
Well I'd just argue it's conceptually different, but alright, it's details